dask_expr/tests/test_shuffle.py

```diff
@@ -55,7 +55,9 @@ def test_disk_shuffle(ignore_index, npartitions, df):
         npartitions=npartitions,
         ignore_index=ignore_index,
     ).partitions[1]
-    assert set(a["x"].compute()).issubset(b["y"].compute())
+    assert set(a["x"].compute().values.tolist()).issubset(
+        b["y"].compute().values.tolist()
+    )
```

Comment:
I know these changes are also annoying, but cudf tries to be loud about device -> host transfers than may be unintentional. We typically need to convert to cupy/numpy first before converting to a list.